src/pages/meetings/index.mdx

```diff
@@ -17,7 +17,7 @@ TSC Meetings
 <div className="container">
 <div className="row">
 <div className="col">
-<p className="text-2xl">The OpenLineage Technical Steering Committee meets monthly on the second Wednesday from 9:30am to 10:30am US Pacific.</p>
+<p className="text-2xl">The OpenLineage Technical Steering Committee meets monthly on the third Wednesday from 9:30am to 10:30am US Pacific.</p>
```

Comment:
@Sheeri Could you please update this file https://github.com/OpenLineage/OpenLineage/blob/main/website/src/pages/meetings/index.mdx instead? openlineage-site is automatically committed from website directory of OpenLineage repo.

Reply:
Will do!